assign_rights/tests.py

```diff
@@ -1,3 +1,38 @@
-# from django.test import TestCase
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIRequestFactory
 
-# Create your tests here.
+from .models import RightsShell
+from .views import RightsAssemblerView
+
+
+class TestModels(TestCase):
+
+    def create_rightsshell(self):
+        return RightsShell.objects.create(
+            rights_id=1,
+            rights_basis="Copyright",
+            copyright_status="copyrighted",
+            determination_date="2020-01-01",
+            note="note",
+            applicable_start_date="2019-02-01",
+            applicable_end_date="2019-03-01",
+            start_date_period=None,
+            end_date_period=None,
+            end_date_open=False,
+            license_terms=None,
+            statute_citation=None
+        )
+
+    def test_rightsshell_creation(self):
+        s = self.create_rightsshell()
+        self.assertTrue(isinstance(s, RightsShell))
```

Comment:
I'm not convinced this test is doing all that much - we're basically just testing that Django's ORM works, which is something that's done much more effectively elsewhere.